CHANGELOG.md

```diff
@@ -21,22 +21,29 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/docker-infrastructure/compare/release-0.3.6...master
+[Unreleased]: https://github.com/atlassian/docker-infrastructure/compare/release-0.3.7...master
 
+## [0.3.7] - 2023-09-11
```

Comment:
Nit: under https://packages.atlassian.com/maven-central/com/atlassian/performance/tools/docker-infrastructure/0.3.7/ I can see it's release date is 2023-09-12

Reply:
Fixed in https://github.com/atlassian/docker-infrastructure/pull/31